public/pages/Snapshots/components/RestoreSnapshotFlyout/RestoreSnapshotFlyout.tsx

```diff
@@ -252,98 +305,109 @@ export default class RestoreSnapshotFlyout extends Component<RestoreSnapshotProp
-            <SnapshotIndicesInput
-              indexOptions={indexOptions}
-              selectedIndexOptions={selectedIndexOptions}
-              onIndicesSelectionChange={this.onIndicesSelectionChange}
-              getIndexOptions={this.getIndexOptions}
-              onCreateOption={this.onCreateOption}
-              selectedRepoValue={repository}
-              isClearable={true}
-            />
-          )}
-
-          <EuiSpacer size="l" />
-
-          <SnapshotRenameOptions
-            doNotRename={renameIndices === do_not_rename}
-            onDoNotRenameToggle={this.onToggle}
-            addPrefix={renameIndices === add_prefix}
-            onAddPrefixToggle={this.onToggle}
-            renameIndices={renameIndices === rename_indices}
-            onRenameIndicesToggle={this.onToggle}
-            width="200%"
-          />
-
-          {renameIndices === add_prefix && <AddPrefixInput getPrefix={this.getPrefix} />}
-          {renameIndices === rename_indices && (
-            <RenameInput getRenamePattern={this.getRenamePattern} getRenameReplacement={this.getRenameReplacement} />
-          )}
-
-          <EuiSpacer size="xxl" />
-          <EuiAccordion id="advanced_restore_options" buttonContent="Advanced options">
-            <EuiSpacer size="m" />
-
-            <SnapshotRestoreAdvancedOptions
-              ignore={snapshot?.ignore_index_settings ? snapshot.ignore_index_settings : false}
-              getIndexSettings={this.getIndexSettings}
-              restoreAliases={String(_.get(snapshot, restore_aliases, true)) == "true"}
-              onRestoreAliasesToggle={this.onToggle}
-              restoreClusterState={String(_.get(snapshot, include_global_state, false)) == "true"}
-              onRestoreClusterStateToggle={this.onToggle}
-              ignoreUnavailable={String(_.get(snapshot, ignore_unavailable, false)) == "true"}
-              onIgnoreUnavailableToggle={this.onToggle}
-              restorePartial={String(_.get(snapshot, partial, false)) == "true"}
-              onRestorePartialToggle={this.onToggle}
-              customizeIndexSettings={String(_.get(snapshot, customize_index_settings, false)) == "true"}
-              onCustomizeIndexSettingsToggle={this.onToggle}
-              ignoreIndexSettings={String(_.get(snapshot, ignore_index_settings, false)) == "true"}
-              onIgnoreIndexSettingsToggle={this.onToggle}
-              width="200%"
-            />
-          </EuiAccordion>
-        </EuiFlyoutBody>
-
-        <EuiFlyoutFooter>
-          <FlyoutFooter edit={true} restore={true} action="" onClickAction={this.onClickAction} onClickCancel={onCloseFlyout} />
-        </EuiFlyoutFooter>
+      <EuiFlyout ownFocus={false} maxWidth={600} onClose={onCloseFlyout} size="m" hideCloseButton>
+        {listIndices && <IndexList indices={indicesList} snapshot={snapshotId} onClick={this.onBackArrowClick} title="Indices in snapshot" />}
+        {listIndices || (
```

Comment:
Not blocking, but could this be simplified to something like the following instead of checking `listIndices` twice?
```
{listIndices 
     ? <IndexList *attributes* /> 
     : (
          <>*the element created on line 314*</>
       )
}

Reply:
yes, makes perfect sense....